src/js/node/fs.ts

```diff
@@ -1076,6 +1078,24 @@ class Dir {
   }
 }
 
+function glob(pattern: string | string[], options, callback) {
+  if (typeof options === "function") {
+    callback = options;
+    options = undefined;
+  }
+  if (typeof callback !== "function") {
+    throw $ERR_INVALID_ARG_TYPE("callback", "function", callback);
+  }
+
+  Array.fromAsync(require("internal/fs/glob").glob(pattern, options ?? kEmptyObject))
+    .then(result => callback(null, result))
+    .catch(callback);
+}
+
+function globSync(pattern: string | string[], options): string[] {
+  return Array.from(require("internal/fs/glob").globSync(pattern, options ?? kEmptyObject));
```

Comment:
definition of `require("internal/fs/glob").globSync` should be moved to file scope